vizro-core/examples/visual-vocabulary/custom_charts.py

```diff
@@ -310,3 +310,59 @@ def diverging_stacked_bar(data_frame: pd.DataFrame, **kwargs) -> go.Figure:
         fig.add_hline(y=0, line_width=2, line_color="grey")
 
     return fig
+
+
+@capture("graph")
+def lollipop(data_frame: pd.DataFrame, **kwargs):
+    """Creates a lollipop based on px.scatter.
+
+    A lollipop chart is a variation of a bar chart where each data point is represented by a line and a dot at the end
+    to mark the value.
+
+    Inspired by: https://community.plotly.com/t/how-to-make-dumbbell-plots-in-plotly-python/47762
```

Comment:
It's a bit weird to me that the `bar` and the `dumbbell` charts referenced, but the `scatter` is used as the "main" chart. Ignore this comment if you disagree 😄

Reply:
The dumbbell reference is wrong! Let me quickly update that 😄 But other than that, I think it's not weird, because you can draw lines and dots with the scatter chart :) And the lollipop is essentially a bar chart alternative.